lib/google/cloud/gemserver.rb

```diff
@@ -22,13 +22,25 @@ module Cloud
     # a gemserver to a Google Cloud Platform project.
     #
     module Gemserver
-      autoload :CLI,           "google/cloud/gemserver/cli"
-      autoload :VERSION,       "google/cloud/gemserver/version"
-      autoload :Configuration, "google/cloud/gemserver/configuration"
-      autoload :StorageSync,   "google/cloud/gemserver/storage_sync"
-      autoload :GCS,           "google/cloud/gemserver/gcs"
-      autoload :Authentication,"google/cloud/gemserver/authentication"
-      autoload :Backend,       "google/cloud/gemserver/backend"
+      autoload :CLI,            "google/cloud/gemserver/cli"
+      autoload :VERSION,        "google/cloud/gemserver/version"
+      autoload :Configuration,  "google/cloud/gemserver/configuration"
+      autoload :GCS,            "google/cloud/gemserver/gcs"
+      autoload :Authentication, "google/cloud/gemserver/authentication"
+
+      ##
+      #
+      # # Backend
+      #
+      # Contains services that run on Google App Engine directly leveraging
+      # tools such as Cloud SQL proxy.
+      #
+      module Backend
```

Comment:
Had a bug previously, but since it has been fixed I agree to decompose `Backend` as you described.